src/components/Inputs/Inputs.js

```diff
@@ -2,141 +2,157 @@ import React, { Component } from 'react';
 
 import { View,
   StyleSheet,
-  Image
+  Image,
+  Text
 } from 'react-native';
 
+import inputImages from '../../img/inputs/index';
+
 export default class Inputs extends Component {
   constructor() {
     super();
     this.state = {};
   }
 
   renderInputs = (inputs) => {
-    // TODO: figure out the best way to implement keys here so RN will stop bitching
     return inputs.map((input, index) => {
       switch (input) {
-        // TODO: add image sources here
+        // TODO: fix up missing image sources
         case '1':
           return (
```

Comment:
Can you explain further? It sounds like you're just re-iterating? Streamlining as in?